esnext.js

```diff
@@ -29,11 +33,18 @@ def.ecmaFeatures = {
 
 def.env.es6 = true;
 def.rules['no-var'] = 2;
-def.rules['object-shorthand'] = [2, 'always'];
 def.rules['prefer-arrow-callback'] = 2;
 def.rules['prefer-const'] = 2;
 def.rules['prefer-reflect'] = [2, {exceptions: ['delete']}];
 def.rules['prefer-template'] = 2;
 def.rules['prefer-spread'] = 2;
+def.rules['babel/object-shorthand'] = [2, 'always'];
+def.rules['babel/generator-star-spacing'] = def.rules['generator-star-spacing'];
+def.rules['babel/arrow-parens'] = def.rules['arrow-parens'];
+def.rules['babel/object-curly-spacing'] = def.rules['object-curly-spacing'];
+def.rules['object-shorthand'] = 0;
+def.rules['generator-star-spacing'] = 0;
+def.rules['arrow-parens'] = 0;
+def.rules['object-curly-spacing'] = 0;
```

Comment:
Mainly to support `async`/`await`, but see [eslint-plugin-babel readme](https://github.com/babel/eslint-plugin-babel#rules) for more reasons.